CHANGELOG.md

```diff
@@ -15,14 +15,71 @@ For reference, the possible headings are:
 * **External Contributors** to list contributors outside of BigchainDB GmbH.
 * **Notes**
 
+## [1.0.0rc1] - 2017-06-??
+Tag name: v1.0.0rc1
+
+### Added
+* Support for SSL/TLS auth and communications between MongoDB and BigchainDB, MongoDB Backup Agent, MongoDB Monitoring Agent, and MongoDB Cloud Manager. Pull Requests
```

Comment:
This should be `Support for secure TLS/SSL communication between MongoDB and BigchainDB, MongoDB Backup Agent, MongoDB Monitoring Agent`.
1. There is no TLS between MongoDB instance and Cloud Manager directly.
2. There is no `auth` support till we merge #1552. We are blocked on Cloud Manager backups using auth.